### ISSUE TITLE
Settubgs trim_trailing_whitespace to false is ignored

### ISSUE THREAD

**Issue by stopiccot** [dotnet/format#532]
I've configured trim_trailing_whitespace to false in my .editorconfig file but running dotnet-format still removes trailing whitespace characters.

dotnet-format version: 3.2.107702+3ecea99de4bb82b724bf11134279b5aaf8dd1f2f

**Comment by JoeRobich** [comment]
You are right. We've had a email chain this week discussing that the Roslyn Formatter always trims trailing whitespace. I'm going to move this to the Roslyn repo.
